### ISSUE TITLE
Same GPU (RTX 5090), same dataset, same settings: Windows 11 works, Linux goes OOM immediately regardless of batch size

### ISSUE THREAD

**Issue by Parogar** [kohya-ss/musubi-tuner#526]
I've got a dataset of about 95 images set for 800x800 resolution with 2 repeats. Trying to train WAN 2.2 high noise. Using the EXACT same config file  with the exact same settings and the EXACT same run conditions, my debian machine will OOM **immediately** even if I reduce the batch size to 1, and my Windows machine can do up to batch size 4.

 resolution, caption_extension, batch_size, num_repeats, enable_bucket, bucket_no_upscale should be set in either general or datasets
# otherwise, the default values will be used for each item

# general configurations
[general]
resolution = [800, 800]
caption_extension = ".txt"
batch_size = 1
enable_bucket = true
bucket_no_upscale = false

[[datasets]]
image_directory = "aw"
cache_directory = "aw/latents"
num_repeats = 2 # optional, default is 1. Number of times to repeat the dataset. Useful to balance the multiple datasets with different sizes.

[[datasets]]
image_directory = "ld"
cache_directory = "ld/latents"
num_repeats = 1

accelerate launch --num_cpu_threads_per_process 4 --mixed_precision fp16 wan_train_network.py --task t2v-14B --dit /mnt/sdb/wan2.2_t2v_high_noise_14B_fp16.safetensors --dataset_config aw.toml --sdpa --mixed_precision fp16 --fp8_base --optimizer_type adamw8bit --learning_rate 1.6e-4 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --timestep_sampling shift --discrete_flow_shift 3.0 --max_train_epochs 32 --save_every_n_epochs 1 --seed 123456--output_dir awlora --output_name Wan2_2_aw_HighNoise --vae /root/ComfyUI/models/vae/wan_2.1_vae.safetensors --t5 models_t5_umt5-xxl-enc-bf16.pth --save_state

**Comment by FurkanGozukara** [comment]
share pip freeze of both platforms

**Comment by kohya-ss** [comment]
I'm not familiar with training in a Linux environment, but it may be because shared GPU memory is not available on Linux. Check the memory usage in Task Manager on Windows, and if shared memory is being used, that may be the cause.

**Comment by Parogar** [comment]
> I'm not familiar with training in a Linux environment, but it may be because shared GPU memory is not available on Linux. Check the memory usage in Task Manager on Windows, and if shared memory is being used, that may be the cause.


You're right. I had no idea that was even a thing.  I apologize for wasting your time. Love all the work you do ty.
